src/ga4gh/vrs/extras/translator.py

```diff
@@ -580,7 +586,7 @@ def _to_hgvs(self, vo, namespace="refseq"):
             if ns.startswith("GRC") and namespace is None:
                 continue
 
-            if not (any(a.startswith(pfx) for pfx in ("NM", "NP", "NC", "NG"))):
+            if not (any(a.startswith(pfx) for pfx in ("NM", "NP", "NC", "NG", "NR", "NW"))):
```

Comment:
What are NW accessions? I found one associated with a linear Zebrafish reference sequence, but have a hard time finding documentation on what this prefix means and how it should be interpreted.

Reply:
https://hgvs-nomenclature.org/stable/background/refseq/?h=nw#recommended-reference-sequences-types-are

> genomic contigs or scaffolds - NT_010718.17, NW_003315950.2

Reply:
there are 182 hgvs expressions in clinvar that have NW_ associated expressions.  Here are a couple if it helps to discern what they are...
https://www.ncbi.nlm.nih.gov/clinvar/variation/146167
(patch scaffolding?)

Reply:
@larrybabb should we also add "NT"?

Reply:
I missed adding the NT, so i posted a new issue.